code/Model/Recipient/EmailRecipientCondition.php

```diff
@@ -96,6 +97,11 @@ public function matches($data)
                     $result = !($result);
                 }
                 break;
+            case 'Includes':
+                $result = is_array($fieldValue)
+                    ? in_array($conditionValue, $fieldValue)
+                    : stripos($fieldValue, $conditionValue) !== false;
```

Comment:
```suggestion
                    : stripos($fieldValue ?? '', $conditionValue) !== false;
```

Conditions can take the whole page and probably other non-fields. Probably that's technically a separate bug, but with the other conditions it doesn't cause a problem. With this new condition, that results in the following error:

![error example](https://github.com/silverstripe/silverstripe-userforms/assets/36352093/184d58a4-f235-49c5-9bb1-55f6bbfaa7f1)

Using the null coalescing operator to convert the null to an empty string resolves that problem.